Dockerfile

```diff
@@ -5,6 +5,8 @@ RUN mkdir -p /srv/node
 ADD package.json /srv/node/
 WORKDIR /srv/node
 
+COPY docker/yarnpkg.gpg.key /tmp/yarnpkg.gpg.key
```

Comment:
I think it would make sense to put the key somewhere long-lived rather than `tmp`. Also, I find it better for sanity if the local `docker` directory mirrors the server's filesystem. I suggest this:

```
COPY docker/etc/pki/yarnpkg.gpg.key /etc/pki/yarnpkg.gpg.key
```

Reply:
It is only useful once, at docker build-time, then it is in `apt`. That's why I put it in `/tmp`, but I can change that.